lib/decompress/zstd_decompress_internal.h

```diff
@@ -236,5 +236,14 @@ size_t ZSTD_loadDEntropy(ZSTD_entropyDTables_t* entropy,
  *  This function cannot fail. */
 void ZSTD_checkContinuity(ZSTD_DCtx* dctx, const void* dst, size_t dstSize);
 
+/*! ZSTD_findFrameCompressedSize_advanced() :
```

Comment:
Why does it need to be published ?
Could it remain a `static` unit-internal function ?